src/main.c

```diff
@@ -118,6 +120,13 @@ static bool parse_args(int argc, char **args)
             opt_bootargs = optarg;
             emu_argc++;
             break;
+        case 'x':
+            if (!strncmp("vblk:", optarg, 5))
```

Comment:
Why must we introduce such complexity with the `vblk:` prefix? The brilliant way is to use just the disk file name here and consider adding the long option for different block devices if required.

Reply:
> Why must we introduce such complexity with the `vblk:` prefix?

VirtIO features like disk, network, and sound can be dynamically enabled using the `-x` option, inspired by [Oracle/Sun Java virtual machine](https://www.oracle.com/java/technologies/javase/vmoptions-jsp.html). For example: `-x vblk:<disk> -x vnet:tap0 -x vsnd:alsa`. The long command options might be confusing if we want to specify more VirtIO devices, such as a secondary virtio-blk.

Reply:
Okay. Thanks